app/models/encounter.rb

```diff
@@ -10,14 +10,19 @@ class Encounter < ActiveRecord::Base
   has_many :test_results, dependent: :restrict_with_error
 
   belongs_to :institution
+  belongs_to :site
   belongs_to :patient
 
   validates_presence_of :institution
+  validates_presence_of :site, if: Proc.new { |encounter| encounter.institution && !encounter.institution.kind_manufacturer? }
```

Comment:
Manufacturers shouldn't need to create Encounters. Moving this to another ticket for now.